native/game_backend/src/player.rs

```diff
@@ -327,6 +330,14 @@ impl Player {
     pub fn inventory_take_at(&mut self, inventory_at: usize) -> Option<Loot> {
         self.inventory[inventory_at].take()
     }
+
+    pub fn can_move(&self) -> bool {
+        self.action.first().is_none()
+    }
+
+    pub fn can_activate(&self) -> bool {
+        self.action.first().is_none()
+    }
```

Comment:
These are just the same behaviour with different name, right?

Reply:
They're exactly the same yeah, we should delete one of them